### DISCUSSION TITLE
User welcome page shrinks

### DISCUSSION THREAD

**Discussion by ghost** [community/community#24633] [Apps, API and Webhooks]
<p>as I was using github on my mobile, I recently noticed the page shrinks a bit from the right (screenshot will be provided) when I go into the welcome page.</p>

**Comment by jfullstackdev** [comment]
<p>yup , can you provide the screenshots</p>

**Comment by ghost** [comment]
<p><a class="mention" href="/jdevstatic">@jdevstatic</a>  the right side there shouldn’t be empty<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/6/96f3954b80ddfcf61d6b75d34a1287dff06d5c3f.png" data-download-href="/uploads/short-url/lxnuHhx8oliLeJbDSm4euuRJy0f.png?dl=1" title="Screenshot_20220403-150332" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10137/181085750-8f1644ed-261c-43c7-8322-d843a39e055c.png" alt="Screenshot_20220403-150332" data-base62-sha1="lxnuHhx8oliLeJbDSm4euuRJy0f" width="241" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96f3954b80ddfcf61d6b75d34a1287dff06d5c3f_2_241x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96f3954b80ddfcf61d6b75d34a1287dff06d5c3f_2_361x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96f3954b80ddfcf61d6b75d34a1287dff06d5c3f_2_482x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96f3954b80ddfcf61d6b75d34a1287dff06d5c3f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screenshot_20220403-150332</span><span class="informations">1074×2226 62.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you mean, it’s not centered ?</p>
<p>you may want to check this out, they are the product team</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback/discussions/categories/mobile-feedback">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback/discussions/categories/mobile-feedback" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181085751-5f010f20-180d-43f7-b4a9-694afd240aea.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/9/d9a1fcffe6b55daa4c800c51812f1abec0db228d_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/9/d9a1fcffe6b55daa4c800c51812f1abec0db228d_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/9/d9a1fcffe6b55daa4c800c51812f1abec0db228d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/9/d9a1fcffe6b55daa4c800c51812f1abec0db228d_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions/categories/mobile-feedback" target="_blank" rel="noopener">Discussions · github/feedback</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - Discussions · github/feedback</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>if there is no similar, you’re always free to post</p>

**Comment by ghost** [comment]
<p>moved the discussion there, thanks!</p>
<p>if this is the case, what is the goal of this community site?</p>

**Comment by jfullstackdev** [comment]
<p>it’s the product team,</p>
<p>this, on the other hand, is more of a forum, you know those lengthy topics, help in coding problems, and others to share expertise, and troubleshooting, I believe so</p>
